tests/plugins/cloudservices/manual/errors.md

```diff
@@ -0,0 +1,12 @@
+@bender-tags: 4.8.0, feature, 932
+@bender-ui: collapsed
+@bender-ckeditor-plugins: sourcearea, wysiwygarea, toolbar, cloudservices
+
+1. Open web inspector.
```

Comment:
Maybe you could change it to `Open browser console.` as web inspector is  usually related with a tab/view responsible for inspecting elements.